common.gypi

```diff
@@ -186,10 +186,12 @@
             ],
           },
           'VCLinkerTool': {
-            'LinkTimeCodeGeneration': 1, # link-time code generation
```

Comment:
Why is this removed?

Reply:
It adds `/LTCG` to the linker options, we want to use `/LTCG:INCREMENTAL`. Right now `gyp` does not support this option directly, so we add it through additional options.